episodes/22-external.md

```diff
@@ -44,14 +70,43 @@ Spend some time doing **one of the following**:
 
```

Comment:
```suggestion

Groups of collaborators taking this training together should discuss first how they will assign these tasks between them.
```

Reply:
The above is intended to make clear how this exercise should be tackled by groups of people working on the same lesson.